samples/python/agents/semantickernel/agent.py

```diff
@@ -296,9 +305,11 @@ async def _ensure_thread_exists(self, session_id: str) -> None:
         Args:
             session_id (str): Unique identifier for the session.
         """
-        if self.thread is None or self.thread.id != session_id:
+        # Replace check with self.thread.id when
+        # https://github.com/microsoft/semantic-kernel/issues/11535 is fixed
+        if self.thread is None or self.thread._thread_id != session_id:
```

Comment:
This has already been addressed and is fixed in main. Please pull in the changes.